docs/source/design.rst

```diff
@@ -71,17 +71,17 @@ A pseudopotential family can be constructed manually, by first constructing the
     from aiida import plugins
 
     UpfData = plugins.DataFactory('pseudo.upf')
-    PseudoPotentialFamily = plugins.GroupFactory('pseudo')
+    PseudoPotentialFamily = plugins.GroupFactory(entry_point_name='pseudo.family')
 
     pseudos = []
 
     for filepath in ['Ga.upf', 'As.upf']:
         with open(filepath, 'rb') as stream:
-            pseudo = UpfData(stream)
-            pseudos.append(pseudo)
+            pseudo = UpfData(file=stream)
```

Comment:
This isn't necessary either I am pretty sure
```suggestion
            pseudo = UpfData(stream)
```

Reply:
The same here. Although, I forgot to add the .store() method in the pseudo, which without it creates an error. The working example should be: pseudos.append(pseudo.store())

Reply:
Should be fine now. Thanks